lib/galaxy/jobs/runners/kubernetes.py

```diff
@@ -4,7 +4,6 @@
 
 import logging
 import math
-import os
```

Comment:
os is used again https://github.com/galaxyproject/galaxy/blob/5598305f07ddde4873a0943c9e5cea9a1f32f94f/lib/galaxy/jobs/runners/kubernetes.py#L161, which was added in the meantime.